src/Connector/SQS.php

```diff
@@ -95,6 +109,19 @@ public function setQueueOptions(array $queueOptions) : void
         }
     }
 
+    /**
+     * @param string[] $message
```

Comment:
This this type hint correct?

Reply:
Nope :( Should be array[] - I'll fix it :+1: - Just writing some tests around this

Reply:
Ah yeah sorry, this one is correct, the parameters for the batch message function wasn't - it should be `string[][] $messages`  which I've updated :)